Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```diff
@@ -186,7 +186,7 @@
 - type: entity
   parent: DrinkBottleGlassBaseFull
   id: DrinkGinBottleFull
-  name: Griffeater Gin
```

Comment:
This sounds like it was intended to be a proper noun, and proper nouns should remain capitalized.